### DISCUSSION TITLE
Switching screens should reset focus item back to first line

### DISCUSSION THREAD

**Discussion by DanielW0830** [VasilKalchev/LiquidMenu#68] [Ideas]
I noticed the menu seems to remember if an item is in focus.
when switching to a new screen, the option to have it reset back to first line would be good.

I'd do it in code, but I have no method which tells me second line is in focus.

**Comment by VasilKalchev** [comment]
You are right, I'll add accessors for the focus variable.

**Comment by VasilKalchev** [comment]
These are the new methods:
```c++ 
/// Directly select focused line.
/**
@param lineIndex - index of the focused line
@return true: on success, false: when the selected line doesn't exist or it isn't focusable
*/
bool set_focusedLine(uint8_t lineIndex);

/// Get the index of the currently focused line.
/**
@returns the index of the currently focused line
*/
uint8_t get_focusedLine() const;
```
They apply to classes: `LiquidScreen`, `LiquidMenu` and `LiquidSystem`.

The update is not yet released, download it from the master branch.
